gradle.properties

```diff
@@ -7,12 +7,12 @@ enigma_version=2.4.1
 unpick_version=2.3.0
 cfr_version=0.2.2
 name_proposal_version=0.2.0
-asm_version=9.7
+asm_version=9.6
 
 # Javadoc generation/linking
 fabric_loader_version=0.15.6
 jetbrains_annotations_version=24.1.0
-mappingpoet_version=0.4.1
+mappingpoet_version=0.3.2
```

Comment:
The MP update is required, just roll back the ASM update. I think we need to get to the bottom of this, as its likely going to be an issue elsewhere.

Reply:
```suggestion
mappingpoet_version=0.4.1
```